### PR TITLE
Add wrap definitions for loguru 2.1.0

### PR DESCRIPTION
**Pull request by nnooney**

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -0,0 +1,67 @@
+project('loguru', 'cpp',
```
Comment (legeana) [inline meson.build]: Could you please use `emilk-loguru` for consistency?

File: meson.build [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+loguru_lib = library(
+  'loguru',
+  files(
+    'loguru.cpp'
+  ),
+  include_directories: include_directories('.'),
+  cpp_args: flags,
+  dependencies: loguru_deps,
+)
+loguru_dep = declare_dependency(
```
Comment (legeana) [inline meson.build]: `emilk_loguru_dep`.

**Review by legeana — CHANGES_REQUESTED** [review]

File: meson.build (hunk 1 above)
Comment (nnooney) [reply meson.build]: Done.

File: meson.build (hunk 2 above)
Comment (nnooney) [reply meson.build]: Done.

**Pull request merged**